corehq/apps/sms/tasks.py

```diff
@@ -604,3 +610,25 @@ def queued_sms():
 
 
 datadog_gauge_task('commcare.sms.queued', queued_sms, run_every=crontab())
+
+if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
+    @periodic_task(run_every=crontab(day_of_month='2', minute=0, hour=0))
+    def send_monthly_sms_report():
+        start_date = date.today().replace(day=1) - relativedelta(months=1)
+        _, last_day = calendar.monthrange(start_date.year, start_date.month)
+        end_date = start_date.replace(day=last_day)
+        filename = call_command('get_icds_sms_usage', 'icds-cas', str(start_date), str(end_date))
+        with open(filename, 'rb') as f:
+            cached_download = expose_cached_download(f.read(), expiry=24 * 60 * 60, file_extension='xlsx')
```

Comment:
rather than storing this in the cache can we store it in the blobldb and use `expose_blob_download`

Reply:
The file is pretty small so I thought of keeping it in cache itself. Not sure it's worth storing and then retrieving. Any benefits by using `expose_blob_download` for this?